.github/workflows/odh-kfto-sdk-notebooks-sync.yaml

```diff
@@ -0,0 +1,170 @@
+# The aim of this GitHub workflow is to update the pipfile to sync with Codeflare-SDK release.
+name: Sync ODH-notebooks with Kubeflow-Training SDK release
+on:
+  workflow_dispatch:
+    inputs:
+      upstream-notebooks-repository-organization:
+        required: true
+        description: "Owner of target upstream notebooks repository used to open a PR against"
+        default: "opendatahub-io"
+      notebooks-target-branch:
+        required: true
+        description: "Target branch of upstream repository"
+        default: "main"
+      python-version:
+        required: true
+        description: "Provide the python version to be used for the notebooks"
+        default: "3.11"
+      notebooks-repository-organization:
+        required: true
+        description: "Owner of origin notebooks repository used to open a PR"
+      training-sdk-release-version:
+        required: true
+        description: "Provide version of the kubeflow-training-sdk release"
+      notebooks-repository-organization-owner-user-email:
+        required: true
+        description: "Provide notebooks repository organization owner's user-email used to open a PR"
+      notebooks-repository-organization-owner-user-name:
+        required: true
+        description: "Provide notebooks repository organization owner's user-name used to open a PR"
```

Comment:
Is it possible to use artificial values here?
Less parameters mean easier action.